data/xiaomitv-ads

```diff
@@ -1,13 +1,14 @@
-ad.intl.xiaomi.com @ads
 ad.mi.com @ads
 ad.xiaomi.com @ads
-admob.xiaomi.com @ads
 ad1.xiaomi.com @ads
+admob.xiaomi.com @ads
 mistat.xiaomi.com @ads
+ad.intl.xiaomi.com @ads
```

Comment:
sorted by alphabet pls

Reply:
i have sorted domains of xiaomi by alphabet and leave last 4 third-party ads domain as they are.